config/resources/publicatie-domain.json

```diff
@@ -134,6 +118,14 @@
           "type": "string",
           "predicate": "dct:title"
         },
+        "due-date": {
+          "type": "datetime",
+          "predicate": "tmo:dueDate"
+        },
+        "target-end-date": {
+          "type": "datetime",
+          "predicate": "tmo:targetEndDate"
+        },
```

Comment:
target-end-date hoort hier niet volgens ticket.

Reply:
maar wel volgens model ...  Lijkt geen gebruik te hebben in huidige flow. Er is volgens mij geen "gevraagde vertaaldatum", dit is altijd de "uiterste vertaaldatum" 
(ook al zijn er designs van de overzicht tabel met gevraagde vertaaldatum maar de data dat erin zit is uiterste vertaaldatum.
Verwarrend